### PR TITLE
Dev: Docs minor improvement

### PR DESCRIPTION
**Pull request by ccamacho**
This patch adds a docs reference
for the commit's style convention.

Please enter the commit message for your changes. Lines starting

### CODE REVIEW

**Review by rbrady — APPROVED** [review]
LGTM.  One suggestion is to change the non-title case title from "Commit's information" to "Commit Messages".

File: docs/src/devel/contributing.rst [hunk 1]
```diff
@@ -164,6 +164,51 @@ data. A common pattern is validating the data file created by the
 associated export role, reading the data file and then calling the
 module you created for import.
 
+
+Commit's messages
```
Comment (rbrady) [inline docs/src/devel/contributing.rst:168]: s/Commit's messages/Commit Messages/

There should not be possessiveness of the messages to the commit. :)  Also, title case.

**Review by rbrady — CHANGES_REQUESTED** [review]

File: docs/src/devel/contributing.rst [hunk 2]
```diff
@@ -164,6 +164,51 @@ data. A common pattern is validating the data file created by the
...
+    |       +-> Summary in present tense.
+    |
+    +-------> Type: [Nn]ew, [cC]hg, [fF]ix.
+
+    <body> ----> The commit's body.
+
+Accepted types:
+
+- `new` or `New`: newly implemented features
+- `chg` or `Chg`: changes in the CI automation, documentation or any other change not presented as a new feature
```
Comment (jistr) [inline docs/src/devel/contributing.rst:191]: I'd say CI automation should have a 'dev:' commit as it's not end-user facing. But this is just a nit. I think we can merge and tweak subsequently as we review docs.

**Review by jistr — APPROVED** [review]

File: docs/src/devel/contributing.rst (hunk 2 above)
Comment (ccamacho) [reply docs/src/devel/contributing.rst:191]: Cool we just need to adjust the changelog config when we update the docs so they are consistent, this patch covers only what we already have

**Pull request merged**